crates/pairwise_nostd_internal/src/apply_points.rs

```diff
@@ -125,17 +95,12 @@ pub fn apply_accum(
     reducer: &impl Reducer,
     points_a: &PointProps,
     points_b: Option<&PointProps>,
-    squared_distance_bin_edges: &[f64],
+    squared_distance_bin_edges: &impl BinEdges,
```

Comment:
Just a comment: I suspect that we probably want to have regularly spaced distance bin edges rather than regularly spaced squared distance bin edges. 

To be clear, I'm not suggesting you change anything for this PR, but maybe we should add a TODO comment?

I'm not even sure how we would elegantly address this... Do you have any ideas?

Reply:
A third `Bins` implementation is what I was imagining. Thoughts?